simulationTool/locales/en/additional.json

```diff
@@ -15,26 +17,45 @@
+        "createEnsembleContent2": "As soon as you click on the plus symbol, you can select the models that your ensemble should contain. Your model selection is displayed in an overview from which you can easily remove models. You can find further settings under the \"Configure\" function. Click on “Simulate ensemble” to calculate the displayed number of scenarios from the models. These scenarios are displayed in the scenario overview.",
+        "createEnsembleContent3": "The model-dependent configurations consist on the one hand of the display of the model uncertainties and on the other hand of the settings of the number of scenarios to be calculated and the selection method of the samples from the uncertainty ranges.",
+        "createEnsembleHint": "Models contain unavoidable mathematical uncertainties that comprise a numerical range instead of a clear numerical value. When an ensemble is simulated, various input parameters from these uncertainty ranges are automatically used to calculate the scenarios.",
+        "createScenario": "Create scenario",
+        "createScenarioContent": "The heart of the Scenario Explorer lies in the execution of simulations. On this page, you can create your own scenario by naming it individually and setting the required input parameters using a wide range of input options. Click on \"Run scenario\" to start the calculations and you will be automatically redirected to the scenario detail page.",
+        "createScenarioHint": "By determining the input parameters, the limiting circumstances under which the respective model is simulated are set. Due to various types of uncertainties, it makes sense to run as many “what-if” scenarios as possible in addition to the “best case”, “worst case” and “business as usual” in order to cover and test a wide range of possible futures that contribute to more robust decision-making.",
         "date": "Date",
+        "definitions": "Definitions",
         "defaultConfirmText": "Are you sure you want to proceed?",
         "description": "Description",
         "ensemble": "Ensemble",
         "ensembleDescription": "Ensemble description",
         "ensembleDetails": "Ensemble details",
+        "ensembleDetailsContent": "You can compare the result values within an ensemble using a line diagram. You can add further scenarios by clicking on the plus symbol so that the values are included in the diagram for comparison. The scenarios included are displayed by name and color-coded according to their associated model. You can also easily remove scenarios from this overview. You can also share the results of the ensemble with other users by clicking on the share symbol and enter into an exchange with them.",
```

Comment:
```suggestion
        "ensembleDetailsContent": "You can compare the result values within an ensemble using a diagram. You can add further scenarios by clicking on the plus symbol so that the values are included in the diagram for comparison. The scenarios included are displayed by name and color-coded according to their associated model. You can also easily remove scenarios from this overview. You can also share the results of the ensemble with other users by clicking on the share symbol and enter into an exchange with them.",
```